src/v/cloud_storage_clients/s3_client.cc

```diff
@@ -401,7 +404,13 @@ ss::future<ResultT> parse_head_error_response(
             code = fmt::format("{}", status_to_error_code(hdr.result()));
             msg = ss::sstring(hdr.reason().data(), hdr.reason().size());
         }
-        auto rid = hdr.at(aws_header_names::x_amz_request_id);
+        std::string_view rid;
```

Comment:
This is a `boost::beast::string_view` I think

Reply:
🤦‍♂️